src/firebase/userDb.js

```diff
@@ -1,64 +1,95 @@
-		user = doc.data();
-	});
-	return user;
+  const ref = query(
+    collection(db, "users"),
+    where("uid", "==", userId)
+  ).withConverter(userConverter);
+  const docSnap = await getDocs(ref);
+  let user;
+  docSnap.forEach((doc) => {
+    // Convert to User object
+    user = doc.data();
+  });
+  return user;
 }
 
 // https://firebase.google.com/docs/firestore/manage-data/add-data
 export async function postUser(user) {
-	try {
-		const ref = collection(db, 'users').withConverter(userConverter);
-		await addDoc(ref, user);
-	} catch (e) {
-		console.error('Error adding document: ', e);
-	}
+  try {
+    const ref = collection(db, "users").withConverter(userConverter);
+    await addDoc(ref, user);
+  } catch (e) {
+    console.error("Error adding document: ", e);
+  }
 }
 
 export async function updateUser(user) {
-	try {
-		const ref = query(collection(db, 'users'), where('uid', '==', user.uid)).withConverter(userConverter);
-		const querySnapshot = await getDocs(ref);
-		let userId;
-		querySnapshot.forEach((doc) => {
-			// doc.data() is never undefined for query doc snapshots
-			userId = doc.id;
-		});
+  try {
+    const ref = query(
+      collection(db, "users"),
+      where("uid", "==", user.uid)
+    ).withConverter(userConverter);
+    const querySnapshot = await getDocs(ref);
+    let userId;
+    querySnapshot.forEach((doc) => {
+      // doc.data() is never undefined for query doc snapshots
+      userId = doc.id;
+    });
 
-		const updateRef = doc(db, 'users', userId).withConverter(userConverter);
-		await setDoc(updateRef, user);
-	} catch (e) {
-		console.error('Error updating document: ', e);
-	}
+    const updateRef = doc(db, "users", userId).withConverter(userConverter);
+    await setDoc(updateRef, user);
+  } catch (e) {
+    console.error("Error updating document: ", e);
+  }
 }
 
 const userConverter = {
-	toFirestore: (user) => {
-		return Object.assign({}, user);
-	},
-	fromFirestore: (snapshot, options) => {
-		const data = snapshot.data(options);
-		return Object.assign({}, data);
-	},
+  toFirestore: (user) => {
+    return Object.assign({}, user);
+  },
+  fromFirestore: (snapshot, options) => {
+    const data = snapshot.data(options);
+    return Object.assign({}, data);
+  },
 };
 
 export async function postMockProperty() {
-	const property = property_json;
-	for (const key in property) {
-		try {
-			const docRef = await addDoc(collection(db, 'properties'), property[key]);
-			console.log('Document written with ID: ', docRef);
-		} catch (e) {
-			console.error('Error adding document: ', e);
-		}
-	}
+  const property = property_json;
+  for (const key in property) {
+    try {
+      const docRef = await addDoc(collection(db, "properties"), property[key]);
+      console.log("Document written with ID: ", docRef);
+    } catch (e) {
+      console.error("Error adding document: ", e);
+    }
+  }
+}
+
+export async function postMockUser() {
+  console.log("inside mock user");
```

Comment:
```suggestion
```
just so that there's no debug statements in main

Reply:
Removed